### PR TITLE
[SCV-25] Browser Types

### PR DESCRIPTION
**Pull request by quetcodesfire**
The issue here was that `browse` types weren't being displayed. I added code to look at file extensions and either return `images/{determinedType}` or `images/jpeg` by default.

### CODE REVIEW

**Comment by DrewE84** [comment]
@quetcodesfire Looks good for the most part. The test you wrote for it seems to be failing though when it's expecting a type of `.jpg` but instead gets a `.jpeg`.

**Comment by DrewE84** [comment]
Great, everything looks good

File: search/lib/convert/granules.js [hunk 1]
```diff
@@ -89,6 +89,21 @@ function cmrGranToFeatureGeoJSON (event, cmrGran) {
...
+      assets.browse.type = 'images/png';
+    }
+    if (browseLink.href.includes('.tiff')) {
+      assets.browse.type = 'images.tiff';
+    }
+    if (browseLink.href.includes('.tif')) {
+      assets.browse.type = 'images.tiff';
+    }
+    if (browseLink.href.includes('.raw')) {
+      assets.browse.type = 'images.raw';
```
Comment (matthewhanson) [inline search/lib/convert/granules.js:103]: this should be "images/raw" rather than images.raw and above should be images/tiff rather than images.tiff

Comment (quetcodesfire) [reply search/lib/convert/granules.js:103]: Fixed.

**Pull request merged**